### PR TITLE
PSR-18 is HTTP Client, not HTTP Factories

### PR DESCRIPTION
**Pull request by riveraj**
I was confused as to why there are two HTTP Factories PSRs. This will not stand! 😄

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ As also described in the [PSR Workflow Bylaw][workflow]. The Editor, or editors,
 
 | Num | Title                                | Editor(s)                      |
 |:---:|--------------------------------------|--------------------------------|
-| 18  | [HTTP Factories][psr18]              | Tobias Nylhom                  |
+| 18  | [HTTP Client][psr18]              | Tobias Nylhom                  |
```
Comment (dbu) [inline index.md]: should the table be aligned?

Comment (riveraj) [reply index.md]: Oops, thanks for the catch! Fixed!

**Comment by dbu** [comment]
@michaelcullum the pullapprove thingy seems to not work out. can you somehow restart it? or is that a manual approval process?

**Comment by Jean85** [comment]
Nope it's correct: it's a general file, so it requires an approval from one of us secretaries.

**Pull request merged**